src/gatsby/gatsby-browser.js

```diff
@@ -5,7 +5,32 @@ import "bootstrap/dist/js/bootstrap.bundle"
 import "@fortawesome/fontawesome-free/css/all.min.css"
 import "./src/styles/style.scss"
 
+// Snowplow tracking
+// see https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/quick-start-guide/?platform=browser
+export const onInitialClientRender = () => {
+  if (typeof window.snowplow === 'function') {
+    window.snowplow("newTracker", "rt", "spt.apps.gov.bc.ca", {
+      appId: "Snowplow_standalone",
+      cookieLifetime: 86400 * 548,
+      platform: "web",
+      post: true,
+      contexts: {
+        webPage: true,
+        performanceTiming: true
+      }
+    });
+
+    // ping every 30 seconds after 30 seconds
+    window.snowplow("enableActivityTracking", 30, 30);
+    window.snowplow("enableLinkClickTracking");
+    window.snowplow("trackPageView");
+  }
+};
+
```

Comment:
I think this was being done in gatsby-ssr so it could be controlled by a server-side environment variable which is set here: https://github.com/bcgov/bcparks.ca/blob/main/.github/workflows/publish-gatsby.yaml

I'll see if I can find a way to control this on our local dev environments without loading it on client render